.github/ISSUE_TEMPLATE/propose_new_rule.yml

```
@@ -0,0 +1,88 @@
+name: Propose a new rule
+description: Propose a new rule for the eslint-plugin-testing-library.
+labels: [new rule]
+body:
+  - type: input
+    id: name_for_new_rule
+    attributes:
+      label: Name for new rule
+      description: Suggest a name for the new rule.
+      placeholder: Put your suggestion for the name in here.
+    validations:
+      required: true
+
+  - type: textarea
+    id: description_of_the_new_rule
+    attributes:
+      label: Description of the new rule
+      description: Please describe what the new rule should do.
+      placeholder: The rule should ...
+    validations:
+      required: true
+
+  - type: textarea
+    id: testing_library_feature
+    attributes:
+      label: Testing library feature
+      description: What Testing Library feature does this rule relate to?
+      placeholder: ...
```

I'm not sure what the placeholder text should be here.

---

I guess this one could be fine without a placeholder, but we can put an existing rule description:
```suggestion
      placeholder: Enforce promises from async queries to be handled
```